corehq/apps/api/tests/audit_resources.py

```diff
@@ -75,6 +75,79 @@ def setUpClass(cls):
             }
         ])
 
+    def test_users_in_specified_domain(self):
+        results = self.resource.query(self.domain1_audits.domain, self.domain1_audits.timezone)
+        for result in results:
+            self.assertTrue(result['user'] in self.domain1_audits.logs.keys())
+
+        results = self.resource.query(self.domain2_audits.domain, self.domain1_audits.timezone)
+        for result in results:
+            self.assertTrue(result['user'] in self.domain2_audits.logs.keys())
+
+    def test_queries_first_last_action_time_for_each_user(self):
+        results = self.resource.query(self.domain1_audits.domain, self.domain1_audits.timezone)
+        self.assertListEqual(results, self.domain1_audits.expected_result)
+
+    def test_queries_ordered_by_local_date_and_user(self):
+        results = self.resource.query(self.domain1_audits.domain, self.domain1_audits.timezone)
+        filtered_results = [(result['local_date'], result['user']) for result in results]
+
+        self.assertListEqual(filtered_results, sorted(filtered_results))
+
+    def test_no_repeated_local_date_per_user(self):
+        results = self.resource.query(self.domain1_audits.domain, self.domain1_audits.timezone)
+        seen_user_local_dates = {}
+        for result in results:
+            user = result['user']
+            local_date = result['local_date']
+            if user in seen_user_local_dates:
+                self.assertNotIn(local_date, seen_user_local_dates[user])
```

Comment:
I don't totally understand what's happening here since the structure seems a bit different. A comment could be helpful.

Reply:
I changed the function name and added a comment. Does that clarify it, or could it use further clarification?
[562bdec](https://github.com/dimagi/commcare-hq/pull/32922/commits/562bdec3e853ec1196167a273b5bc7e58ed09509)